src/lib/PreviewUI.js

```diff
@@ -369,11 +369,14 @@ class PreviewUI {
      */
     setupHeader(headerTheme, logoUrl) {
         const headerContainerEl = this.container.querySelector(SELECTOR_BOX_PREVIEW_HEADER_CONTAINER);
-        headerContainerEl.classList.remove(CLASS_HIDDEN);
-
+        const customLogoEl = this.container.querySelector(SELECTOR_BOX_PREVIEW_LOGO_CUSTOM);
         const headerEl = headerContainerEl.firstElementChild;
+        const defaultLogoEl = headerEl.querySelector(SELECTOR_BOX_PREVIEW_LOGO_DEFAULT);
+
+        headerContainerEl.classList.remove(CLASS_HIDDEN);
```

Comment:
Can we restore this line to its prior position? It doesn't seem related to the other changes.